data/ResourceUtil.cpp

```diff
@@ -1,104 +1,106 @@
-//this file was auto-generated by res2h
+// this file was auto-generated by res2h
 
 #include "Resources.h"
 
-const size_t res2hNrOfFiles = 46;
+const uint32_t res2hNrOfFiles = 47;
```

Comment:
By the way, as I stated before, the goal of this first pull request is to give an "as it" version of the gridview, I can add small changes like the xml modifier to make it fancy, but if I had to add bigger changes like allow themes to change the image for games without boxarts (if it wasn't already here), I would prefer to do it in separates pull requests.